src/Microsoft.ML.Transforms/Text/TextFeaturizingEstimator.cs

```diff
@@ -31,10 +31,38 @@ namespace Microsoft.ML.Transforms.Text
     /// </summary>
     public interface IStopWordsRemoverOptions { }
 
-    // A transform that turns a collection of text documents into numerical feature vectors. The feature vectors are counts
-    // of (word or character) ngrams in a given text. It offers ngram hashing (finding the ngram token string name to feature
-    // integer index mapping through hashing) as an option.
-    /// <include file='doc.xml' path='doc/members/member[@name="TextFeaturizingEstimator "]/*' />
+    // Does this estimator produce SkipGram, or just N-Gram?
+    /// <summary>
+    ///  An estimator that turns a collection of text documents into numerical feature vectors.
+    ///  The feature vectors are normalized counts of word and/or character ngrams (based on the options supplied).
+    /// </summary>
+    /// <remarks>
+    ///     <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes. |
+    /// | Input column data type | [text](xref:Microsoft.ML.Data.TextDataViewType) |
+    /// | Output column data type | vector of <xref:System.Single> |
```

Comment:
>vector [](start = 36, length = 6)

Known-sized Vector

Reply:
leaving it, since this is an output. 

---
In reply to: [277114222](https://github.com/dotnet/machinelearning/pull/3438#discussion_r277114222) [](ancestors = 277114222)